hopp/tools/resource/pysam_wind_tools.py

```diff
@@ -0,0 +1,282 @@
+    data_to_field_number = {'temperature': 1, 'pressure': 2, 'speed': 3, 'direction': 4}
+    out = csv_to_dataframe(wind_csv_filepath, resource_height, resource_year)
+    heights = [h for h in out.iloc[4].to_list() if h is not None]
+    field_names = [h for h in out.iloc[2].to_list() if h is not None]
+    field_numbers = [data_to_field_number[f] for f in field_names]
+    data = out.loc[5:].dropna(axis=1)
+    formatted_data = [d.tolist() for d in data.values]
+    wind_resource_data = {
+        'heights':heights,
+        'fields':field_numbers,
+        'data':formatted_data}
+    return wind_resource_data
+
+
+def combine_and_write_srw_files(file_resource_heights, output_filepath):
+
+    data = [None] * 2
+    for height, f in file_resource_heights.items():
+        if os.path.isfile(f):
+            with open(f) as file_in:
+                csv_reader = csv.reader(file_in, delimiter=',')
+                line = 0
+                for row in csv_reader:
+                    if line < 2:
+                        data[line] = row
+                    else:
+                        if line >= len(data):
+                            data.append(row)
+                        else:
+                            data[line] += row
+                    line += 1
+
+    with open(output_filepath, 'w', newline='') as fo:
+        writer = csv.writer(fo)
+        writer.writerows(data)
+
+def combine_wind_resource_data(wind_resource_data):
+    """Combines dictionaries of wind resoure data.
+
+    Args:
+        wind_resource_data (list[dict]): list of wind resource data dictionaries for different resource heights
+
+    Returns:
+        dict: wind resource data dictionary for all hub-heights
+    """
+    all_heights = [wind_resource_data[i]['heights'] for i in range(len(wind_resource_data))]
+    all_fields = [wind_resource_data[i]['fields'] for i in range(len(wind_resource_data))]
+    all_data = [wind_resource_data[i]['data'] for i in range(len(wind_resource_data))]
+    
+    
+    heights = sum(all_heights,[])
+    fields = sum(all_fields,[])
+    data = np.concatenate(all_data,axis=1)
+
+    height_field = [f"{f}-{h}m" for f,h in zip(fields,heights)]
+    if any(height_field.count(ff)>1 for ff in height_field):
+        duplicate_data_entries = [ff for ff in height_field if height_field.count(ff)>1]
+        duplicate_data_entries = list(set(duplicate_data_entries))
+        for drop_data_entry in duplicate_data_entries:
+            if drop_data_entry in height_field:
+                i_drop = height_field.index(drop_data_entry)
+                heights.pop(i_drop)
+                fields.pop(i_drop)
+                height_field.pop(i_drop)
+                data = np.delete(data,i_drop,axis=1)
+
+    combined_resource_data = {
+        'heights':heights,
+        'fields':fields,
+        'data':data.tolist(),
+        }
+    return combined_resource_data
+
+def combine_CSV_to_wind_data(file_resource_heights, fake_resource_year = '1900'):
```

Comment:
What is this `fake_resource_year` used for? If it's just to pass in a year to other functions and has no real meaning, could we change it to `None` and then modify the functions that use it elsewhere? Or maybe even have `None` be the default for those functions and then only use the year if it's not None. Feel free to call me if this isn't making sense, I just want to avoid confusion about why there's a `fake_resource_year`